.github/workflows/clang-format-pull-request.yml

```diff
@@ -19,8 +19,8 @@ jobs:
     - name: Applying clang-format for changed files
       run: |
         URL="https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/${{ github.event.pull_request.number }}/files"
```

Comment:
It's a bit out-of-scope of this PR, but should be considered as well. Using API may introduce failures once it reaches [rate limit](https://docs.github.com/en/free-pro-team@latest/rest/overview/resources-in-the-rest-api#rate-limiting):

> For unauthenticated requests, the rate limit allows for up to 60 requests per hour. Unauthenticated requests are associated with the originating IP address, and not the user making requests.

So I'd propose to consider something like `git diff --name-only  <target-branch> <source-branch>` to get a list of changed files.

Reply:
Not fully sure this will work in case of github checkout action. Taking into account rate limits listed and number of pull requests I think we can postpone this change till start reaching the limit.

Reply:
The limit may be reached even with low number of PRs for this repo in case of re-using the same IP for other API calls in any other repo. But it is just an assumption and since it's out of scope of this PR I resolve it.